gapic-generator/lib/gapic/presenters/gem_presenter.rb

```diff
@@ -191,11 +191,12 @@ def extra_files
       end
 
       def dependencies
-        deps = { "gapic-common" => [">= 0.4.1", "< 2.a"] }
-        deps["grpc-google-iam-v1"] = [">= 0.6.10", "< 2.a"] if iam_dependency?
-        extra_deps = gem_config :extra_dependencies
-        deps.merge! extra_deps if extra_deps
-        deps
+        @dependencies ||= begin
+          deps = { "gapic-common" => [">= 0.4.1", "< 2.a"] }
+          deps["grpc-google-iam-v1"] = [">= 0.6.10", "< 2.a"] if iam_dependency?
+          extra_deps = gem_config_dependencies
+          deps.merge! extra_deps if extra_deps
+        end
```

Comment:
You'll need to specify `deps` at the end of the block. Otherwise, line 198 will return nil if !extra_deps.

Reply:
also fixed